app/templates/cdv-generic-nav-bar.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if nav.leftButton.text}}
   <button {{action 'leftButton'}}>
     {{#if nav.leftButton.icon}}
-      <i {{bind-attr class=":icon nav.leftButton.icon"}}></i>
+      <i class="icon {{nav.rightButton.icon}}"></i>
```

Comment:
Besides being a duplicate, this needs to be `leftButton`, of course. Why not close this?